### PR TITLE
Deprecate sample size

### PR DESCRIPTION
**Pull request by patrickvonplaten**

### CODE REVIEW

File: src/diffusers/modeling_utils.py [hunk 1]
```diff
@@ -448,7 +448,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         if low_cpu_mem_usage:
             # Instantiate model with empty weights
             with accelerate.init_empty_weights():
-                model, unused_kwargs = cls.from_config(
+                config, unused_kwargs = cls.load_config(
```
Comment (patrickvonplaten) [inline src/diffusers/modeling_utils.py:451]: forgot to change this in a previous PR

File: src/diffusers/utils/deprecation_utils.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def deprecate(*args, take_from: Optional[Union[Dict, Any]] = None, standard_warn
 
         if warning is not None:
             warning = warning + " " if standard_warn else ""
-            warnings.warn(warning + message, DeprecationWarning)
+            warnings.warn(warning + message, FutureWarning)
```
Comment (patrickvonplaten) [inline src/diffusers/utils/deprecation_utils.py:35]: We should actually use FutureWarning - it's better and displayed more prominently

File: src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py [hunk 3]
```diff
@@ -132,6 +133,27 @@ def __init__(
                 " checker. If you do not want to use the safety checker, you can pass `'safety_checker=None'` instead."
             )
 
+        is_unet_version_less_0_9_0 = hasattr(unet.config, "_diffusers_version") and version.parse(
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:136]: To make sure by default stable diffusion models use the correct image size

**Review by konqiDAM — COMMENTED** [review]
Ok

**Pull request merged**